src/core/utils/conversation/helpers.ts

```diff
@@ -188,10 +194,10 @@ export const highlightMatch = (
 /**
  * Removes the system prompt from the conversation array since we don't include that when searching
  */
-const removeInitialSystemMessage = (conversation: ConversationHistory) => {
-  return conversation.conversation[0]?.role === 'system'
-    ? conversation.conversation.slice(1)
-    : conversation.conversation;
+const removeSystemMessages = (conversationHistory: ConversationHistory) => {
+  return conversationHistory.conversation.filter(
+    (msg) => msg.role !== 'system',
+  );
```

Comment:
We don't want to include this message when a user searches

![Screenshot 2025-03-24 at 1 02 55 PM](https://github.com/user-attachments/assets/f93a5a68-8ece-4834-ba73-6d1fac151175)